lib/PhasarLLVM/Pointer/LLVMPointsToSet.cpp

```diff
@@ -59,6 +83,95 @@ LLVMPointsToSet::LLVMPointsToSet(ProjectIRDB &IRDB, bool UseLazyEvaluation,
   }
 }
 
+LLVMPointsToSet::LLVMPointsToSet(ProjectIRDB &IRDB,
+                                 const std::string& PointsToSetFile)
+{
+  load(PointsToSetFile, IRDB);
+}
+
+void LLVMPointsToSet::save(const std::string &PointsToSetFile, ProjectIRDB &IRDB) {
```

Comment:
Is there a particular reason why we encode the save/load and stream handling our selfs?
From my point of view, it would be easier to encode the data structure into llvms yaml traits and let them do the heavy lifting.